api/src/services/survey-service.ts

```diff
@@ -1101,4 +1101,37 @@ export class SurveyService extends DBService {
 
     return PublishStatus.SUBMITTED;
   }
+  /**
+   * Get all critter associations for the given survey. This only gets you critter ids, which can be used to fetch details from the external system.
+   * @param {number} surveyId
+   * @returns {*}
+   */
+  async getCrittersInSurvey(surveyId: number) {
```

Comment:
I wonder if maybe we should have a `critter-service`? The survey service is becoming pretty bloated at this point 🤔

Reply:
Agree with this, I will make a separate repository and service class.